app/middleware/getServices.js

```diff
@@ -0,0 +1,75 @@
+const VError = require('verror').VError;
+const constants = require('../lib/constants');
+const elasticsearchClient = require('../lib/elasticsearchClient');
+const esGeoQueryBuilder = require('../lib/esGeoQueryBuilder');
+const esGetServiceHistogram = require('../lib/promHistograms').esGetServices;
+const esQueryLabelName = require('../lib/constants').promEsQueryLabelName;
+const serviceDataMapper = require('../lib/utils/serviceDataMapper');
+const log = require('../lib/logger');
+
+function handleError(error, next) {
+  const errMsg = 'Error with ES';
+
+  log.error({ err: new VError(error.stack, errMsg) }, errMsg);
+  next(error);
```

Comment:
Suggest to pass the `new VError(...)` created in the line above as opposed to the original one without the additional `errMsg` context.